### DISCUSSION TITLE
updating data set and gym to newer coq version

### DISCUSSION THREAD

**Discussion by brando90** [princeton-vl/CoqGym#71] [Q&A]
Hi,

What is the procedure to update the data set & gym? what is the estimated effort to do that? (btw, what version is it using now?)

**Comment by yangky11** [comment]
The Coq version used in this repo is 8.9.1. The main difference with the official Coq 8.9.1 is the code for saving `*.meta` files in [coq/toplevel/vernac.ml](https://github.com/princeton-vl/CoqGym/blob/master/coq/toplevel/vernac.ml). In principle, it should be possible to migrate to newer versions of Coq by making similar changes to Coq, compiling the Coq projects and then extracting the data. But I guess it would take some non-trivial amount of effort and may result in a slightly different dataset. Some Coq projects may not be able to compile using the new version.
